src/app/administrator/administrator.component.html

```diff
@@ -0,0 +1,6 @@
+<p>
+  What would you like to do?
+</p>
+<ul>
+  <li><a routerLink="/lms/admin/loans">Override a book loan due date</a></li>
```

Comment:
I am going to make a quick admin page to handle these links next.

Reply:
I can handle the styling and layout for admin pages, basically just basing it on whatever you do for the home page. Then each of us can say that we did some work on that stuff.

Reply:
Hey I made a rough layout page for the admin. Pretty much like you said, it looks like the home page. I wanted to get all the links around the site done in one go.